pkg/provisioner/ironic/ironic.go

```diff
@@ -1199,14 +1199,35 @@ func (p *ironicProvisioner) ironicHasSameImage(ironicNode *nodes.Node) (sameImag
 	return sameImage
 }
 
-func (p *ironicProvisioner) buildManualCleaningSteps() (cleanSteps []nodes.CleanStep) {
+func (p *ironicProvisioner) buildManualCleaningSteps() (cleanSteps []nodes.CleanStep, err error) {
+	// Build raid clean steps
+	if p.bmcAccess.RAIDInterface() != "" {
+		cleanSteps = append(cleanSteps, buildRAIDCleanSteps(p.host.Status.Provisioning.RAID)...)
+	} else if p.host.Status.Provisioning.RAID != nil {
+		return nil, fmt.Errorf("RAID settings are exist, but the node's RAID interface is <None>")
```

Comment:
This should be reworded to something like ", but the node's driver %s does not support RAID"

Reply:
Thanks, done.